wrapanapi/google.py

```diff
@@ -265,6 +275,20 @@ def get_file_from_bucket(self, bucket_name, file_name):
                     raise error
         return {}
 
+    def delete_file_from_bucket(self, bucket_name, file_name):
+        if self.bucket_exists(bucket_name):
+            try:
+                data = self._storage.objects().delete(bucket=bucket_name,
+                                                      object=file_name).execute()
+                return data
+            except errors.HttpError as error:
+                if "Not Found" in error.content:
+                    self.logger.info(
+                        "File {} was not found in bucket {}".format(bucket_name, file_name))
+                else:
+                    raise error
+        return {}
```

Comment:
**QUESTION:** Do we really need to return anything here?

Reply:
@jamesooden made it return {} to look like get_file_from_bucket

Reply:
I don't know enough about what the caller is expecting.   This allows for someone expecting a dictionary to call this, though its really in essence return a NULL dictionary.   I guess keep this, lacking better guidance.